include/revng/Support/ArtifactLocation.h

```diff
@@ -0,0 +1,162 @@
+#pragma once
```

Comment:
Let's move this to `include/revng/Pipes` if we want this to be specific to revng, or `include/revng/Pipeline` if it's more general.
In `librevngSupport` we don't have any concept such as "artifact".

Reply:
It lives in `Pipeline` now.